### ISSUE TITLE
`chectl` fails to find install download link for Apple silicon - Darwin ARM64

### ISSUE THREAD

**Issue by cgruver** [eclipse/che#22058]
### Describe the bug

The `chectl` install fails to find a download link:

```bash
bash <(curl -sL  https://www.eclipse.org/che/chectl/)
```

Throws the following error:

```bash
ERROR: No download link found at https://che-incubator.github.io/chectl/download-link/stable-darwin-arm
```

### Che version

7.61@latest

### Steps to reproduce

```bash
bash <(curl -sL  https://www.eclipse.org/che/chectl/)
chectl script requires superuser access.
You will be prompted for your password by sudo.
Password:
curl: (22) The requested URL returned error: 404
ERROR: No download link found at https://che-incubator.github.io/chectl/download-link/stable-darwin-arm
```

### Expected behavior

`chectl` install is successful

### Runtime

other (please specify in additional context)

### Installation method

chectl/latest

### Environment

macOS

### Additional context

The `chectl` CLI can be installed by manually downloading and unpacking the tarball from the `chectl` releases links.

**Comment by AObuchow** [comment]
@cgruver Thank you very much for reporting this.

Seems to be a similar error to https://github.com/eclipse/che/issues/21783, though I'm not sure why this issue is occurring this time around.

CC: @tolusha

**Comment by tolusha** [comment]
I see.
We have chectl binaries for arm64 platform [1], but we have an issue with building the correct download url.
Let me fix it. As a workaround you can download binaries manually.

[1] https://github.com/che-incubator/chectl/releases/tag/7.61.0

**Comment by tolusha** [comment]
@cgruver 
Could you try to install from `next` channel?
`bash <(curl -sL  https://www.eclipse.org/che/chectl/) --channel=next`

**Comment by cgruver** [comment]
@tolusha   After installing from Next channel, it was also able to update back to stable.

```bash
chectl update stable
chectl: Updating CLI from 0.0.20230314-next.f7d1fd4 to 7.62.0... done
chectl: Updating CLI... done
```

**Comment by cgruver** [comment]
Actually, it looks like the issue is resolved with the release of `chectl` `7.62.0`
